FileShareService.DesktopClient.Core/Jobs/AppendAclJob.cs

```diff
@@ -1,12 +1,53 @@
-﻿using Prism.Mvvm;
+﻿using Newtonsoft.Json.Linq;
+using Prism.Mvvm;
+using System.Collections.Generic;
+using System.Linq;
 
 namespace UKHO.FileShareService.DesktopClient.Core.Jobs
 {
     public class AppendAclJob : BindableBase, IJob
     {
+        public const string JobAction = "appendAcl";
         public string DisplayName { get; set; }
 
         public AppendAclJobParams ActionParams { get; set; }
+
+        public List<string> ErrorMessages { get ; set ; }
```

Comment:
Please inline initialise `ErrorMessages` to avoid the compiler warning
````
public List<string> ErrorMessages { get ; private set ; } = new List<string>();
````

Reply:
Fixed